swap_meet/vendor.py

```diff
@@ -0,0 +1,62 @@
+# WAVE ONE
+
+class Vendor:
+    def __init__(self, inventory = []):
+        self.inventory = inventory 
```

Comment:
Because of the funny way that python deals with mutable datatypes as default arguments, this syntax will make it so that all instances of the `Vendor` class have `inventory` that points to the same place in memory. Instead, for default mutable datatypes we should do this:
```suggestion
    def __init__(self, inventory = None):
        if self.inventory == None:
            self.inventory = []
        else:
            self.inventory = inventory 
```